src/store/modules/accounts/hdWallet.js

```diff
@@ -15,16 +15,7 @@ export default {
   },
   actions: {
     create({ state, commit }) {
-      commit('accounts/add',
-        {
-          idx: state.nextAccountIdx,
-          color:
-            // eslint-disable-next-line no-bitwise
-            state.nextAccountIdx === 1 ? '#00FF9D' : `#${((Math.random() * 0xffffff) << 0).toString(16)}`,
-          shift: Math.floor(Math.random() * 100),
-          type,
-        },
-        { root: true });
+      commit('accounts/add', { idx: state.nextAccountIdx, type }, { root: true });
```

Comment:
why these changes are relevant?

Reply:
We've removed the previous logic with `shift` and `color` being generated for each account and stored, now we can not store it.

Reply:
tobe specific, why `commit` method is used in here?

Reply:
`The only way to actually change state in a Vuex store is by committing a mutation.`
https://vuex.vuejs.org/guide/mutations.html